Users/templates/registration/login.html

```diff
@@ -87,8 +88,13 @@ <h4 style="font-family: Cormorant Infant">Welcome to ContriHUB</h4>
                             <label for="form4">Password</label>
```

Comment:
please use the descriptive class names, form4 type of names doesn't make sense :smile: